tools/to_ipynb.py

```diff
@@ -0,0 +1,116 @@
+import os
+import nbformat
+from nbformat.v4 import new_code_cell, new_markdown_cell, new_notebook
+import re
+from redbaron import RedBaron
+import yaml
+
+
+with open('samples.yaml', 'r') as f:
+    samples = yaml.load(f.read())
+
+
+def should_concat(prev_type, cur_type):
+    concat_types = ['comment', 'import', 'from_import', 'assignment']
+    import_types = ['import', 'from_import']
+
+    result = False
+
+    if prev_type == cur_type and cur_type in concat_types:
+        result = True
+
+    if prev_type in import_types and cur_type in import_types:
+        result = True
+
+    return result
+
+def py_to_ipynb(root, path, py_filename, git_clone=False, remove={}):
```

Comment:
remove={} returns a dangerous default value warning.

Reply:
changed to `remove=None`, but why is that dangerous?